doc/source/operations/rabbitmq.rst

```diff
@@ -52,11 +75,18 @@ If you are running Wallaby or Xena, synchronise the Pulp containers.
 
   kayobe playbook run etc/kayobe/ansible/pulp-container-sync.yml pulp-container-publish.yml -e stackhpc_pulp_images_kolla_filter=rabbitmq
```

Comment:
Should this be `etc/kayobe/ansible/pulp-container-publish.yml` instead of `pulp-container-publish.yml`?

Reply:
Yes, good spot. Good you create a PR?